index.html

```diff
@@ -0,0 +1,99 @@
+<html>
+    <head>
+        <title>Guillain</title>
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" />
+        <link rel="stylesheet" href="style.css" />
+    </head>
+    <body class="container-fluid">
+        <div style="position: fixed" class="mt-4">
```

Comment:
You need to move all your CSS code in one file

Reply:
### I think it would be best if your comments don't sound like orders as pointed out in the [guidelines](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)

> We have to remember that there is no way for the student to know our "tone" in written format. The best way to give reviews is to be as polite as possible to avoid friction or misunderstandings!